javav2/example_code/timestream/Readme.md

```diff
@@ -0,0 +1,105 @@
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single action
+
+The following examples use the **TimestreamWriteClient** object:
+
+- [Creating a database](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/CreateDatabase.java) (CreateDatabase command)
+- [Creating a database table](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/CreateTable.java) (CreateTable command)
+- [Deleting a database](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/DeleteDatabase.java) (DeleteDatabase command)
+- [Describing a database](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/DescribeDatabase.java) (DescribeDatabase command)
+- [Describing a database table](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/DescribeTable.java) (DescribeTable command)
+- [Listing all databases](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/ListDatabases.java) (ListDatabases command)
+- [Listing all tables](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/ListTables.java) (ListTables command)
+- [Update a database table](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/UpdateTable.java) (UpdateTable command)
+- [Write data into a table](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/timestream
+/src/main/java/com/timestream/write/WriteData.java) (writeRecords command)
+
+
+## Running the Amazon Timestream Java files
+
+**IMPORTANT**
+
+The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details about the charges you can expect for a given service and operation.   
+
+Some of these examples perform *destructive* operations on AWS resources, such as deleting an table. **Be very careful** when running an operation that deletes or modifies AWS resources in your account. It's best to create separate test-only resources when experimenting with these examples.
```

Comment:
an table. --> a table. 

It's best to create --> We recommend creating

Reply:
2x checked